composer.json

```diff
@@ -2,6 +2,7 @@
     "name": "sparkpost/php-sparkpost",
     "description": "SDK for interfacing with SparkPost APIs",
     "license": "Apache 2.0",
+    "version": "0.2.0",
```

Comment:
0.* versions have all kinds of problems, the sooner we get to 1.x the better imo, for everything (npm init defaults to 1.0.0 now because of how volatile and weird the semver rules are around 0.x versions)